src/aiperf/dataset/utils.py

```
@@ -122,3 +122,99 @@ def sample_positive_normal_integer(mean: float, stddev: float) -> int:
+    file_path = Path(filename)
+    check_file_exists(file_path)
+
+    # Determine format from extension
+    suffix = file_path.suffix.lower()
+    if suffix == ".mp4":
+        video_format = VideoFormat.MP4
+    else:
+        raise RuntimeError(
+            f"'{suffix}' is not one of the supported video formats: "
+            f"{', '.join([f.value for f in VideoFormat])}"
+        )
```

Ruff (0.14.2)

Avoid specifying long messages outside the exception class

(TRY003)